pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, goPackages, fetchFromGitHub }:
+
+goPackages.buildGoPackage rec {
+  name = "prometheus-nginx-exporter-${builtins.substring 0 7 rev}";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/doc/coding-conventions.xml#L259
Read this please.